_includes/author.html

```diff
@@ -0,0 +1,28 @@
+{%- assign author = include.author %}
+{%- if author.id %}
+  {%- assign name = nil %}
+  {%- assign link = nil %}
+  {%- assign affiliation = nil %}
+  {%- for group in site.data.organization-members %}
+    {%- for member in group.members %}
+      {%- if author.id == member.id %}
+        {%- assign name = member.name %}
+        {%- assign link = "/o-nas/#" | append: member.id %}
```

Comment:
I wonder if you could write something like
```
{% assign member = site.members | find: "id", author.id %}
```
here instead? (I haven't tested it but it could work.)

Reply:
Already done (in current version).